docs/topics/OAuth2.md

```diff
@@ -56,68 +43,232 @@ Scopes provide access to certain resources of a user's account. Your API client
 
-When a user is directed to this URL, they are prompted to select a channel for the webhook to be placed in. Your application will receive an authorization code back in the querystring (as usual with the authorization code grant).
+If you continue into the full stack flow while including "bot" in your scopes, you'll get some additional querystring parameters on redirection: `guild_id`, the id of the guild to which the bot was added, and `permissions`, the permission integer from the original URL.
 
-When you exchange the authorization code for an access token, the token response will contain the [webhook](#DOCS_WEBHOOK/webhook-object) object:
+## Webhooks
+
+Discord's webhook flow is a specialized version of an [authorization code](#DOCS_OAUTH2/authorization-code-grant) implementation. In this case, the `scope` querystring parameter needs to be set to `webhook.incoming`:
+
+```
+https://discordapp.com/oauth2/authorize?response_type=code&client_id=157730590492196864&scope=webhook.incoming&redirect_uri=https%3A%2F%2Fnicememe.website
+```
+
+When the user navigates to this URL, they will be prompted to select a channel in which to allow the webhook. When the webhook is [executed](#DOCS_WEBHOOK/execute-webhook), it will post it's message into this channel. On acceptance, the user will be redirected to your `redirect_uri`. The URL will contain the `code` querystring parameter which should be [exchanged for an access token](#DOCS_OAUTH2/authorization-code-grant-access-token-exchange-example). In return, you will receive a slightly modified token response:
+
+###### Webhook Token Response Example
 
 ```json
 {
-	"token_type": "Bearer",
-	"access_token": "7r70pJOvarwv1fkPqacZqFOCv39tX2",
-	"scope": "webhook.incoming",
-	"expires_in": 604800,
-	"refresh_token": "TY0U8LP8joJURIhqREL4AuQXcj5DlO",
-	"webhook": {
-		"name": "test",
-		"channel_id": "199737254929760256",
-		"token": "DuAt6zzLQpPhaAq0IcnCrDUWWpY9Y07dqkB5ulLkhwpA00ZK7IjLve5AE4ACUZqCUTY8",
-		"avatar": "eaa0292a003ceb15264a838a8eff961a",
-		"guild_id": "199737254929760256",
-		"id": "236380988341485568"
-	}
+    "token_type": "Bearer",
+    "access_token": "GNaVzEtATqdh173tNHEXY9ZYAuhiYxvy",
+    "scope": "webhook.incoming",
+    "expires_in": 604800,
+    "refresh_token": "PvPL7ELyMDc1836457XCDh1Y8jPbRm",
+    "webhook": {
+        "name": "testwebhook",
+        "url": "https://discordapp.com/api/webhooks/347114750880120863/kKDdjXa1g9tKNs0-_yOwLyALC9gydEWP6gr9sHabuK1vuofjhQDDnlOclJeRIvYK-pj_",
+        "channel_id": "345626669224982402",
+        "token": "kKDdjXa1g9tKNs0-_yOwLyALC9gydEWP6gr9sHabuK1vuofjhQDDnlOclJeRIvYK-pj_",
+        "avatar": null,
+        "guild_id": "290926792226357250",
+        "id": "347114750880120863"
+    }
 }
 ```
 
+You can ignore everything that is not the `webhook` object. Inside this object, what we really care about is `webhook.url`. This is the URL to which you will make POST requests in order to [execute your webhook](#DOCS_WEBHOOK/execute-webhook).
```

Comment:
> You can ignore everything that is not the `webhook` object

i mean, unless you actually want that information to link the webhook to a user.. i don't think we tell people to ignore it, lol